server/nightmare/script.js

```diff
@@ -1,25 +1,32 @@
-var Nightmare = require('nightmare');
-var Promise = require('bluebird');
+//var Nightmare = require('nightmare');
```

Comment:
Once we confirm this all works, we should change the file path to something not including `nightmare`. Created backlog issue #89.